test/integration/spec/connect.js

```diff
@@ -258,6 +258,63 @@ describe('connect', function() {
     });
   });
 
+  describe('default(gll) signaling region on LocalParticipant', () => {
```

Comment:
@PikaJoyce I think its great that you were able to write tests to verify your implementation. However, I just realized there is already a test suite for signaling regions. Please search for `signaling region`, and then just add an `assert.equals` check for the signaling region at the end of the test function.